decompressed/base/etc/init.d/rootdevice

```diff
@@ -1464,7 +1487,7 @@ ledfw_rework_TG799() {
 
 ledfw_rework_TG800() {
 	if [ ! "$(uci get -q button.info)" ]; then
-		logger_command "Extracting ledfw_support-specificTG800.bz2 and setting up wifi button..."
+		logger_command "Extracting ledfw_support-specificTG800.bz2 and setting up status (wifi) button..."
```

Comment:
huh? was not wps button?

Reply:
???

Reply:
Says WiFi

Reply:
idk i don't remember finally how we configured it, but i remember was the wps button and not wifi